### PR TITLE
Legal Information Updates

### PR DESCRIPTION
**Pull request by PokemonGoZ**

### CODE REVIEW

**Comment by tnsbm** [comment]
Can you move it to the bottom? We already have Licensing part there

**Comment by PokemonGoZ** [comment]
@tnsbm Done.

File: README.md [hunk 1]
```diff
@@ -44,9 +44,11 @@ Outpox
 [PokemonGo-DesktopMap](https://github.com/mchristopher/PokemonGo-DesktopMap) Amazing job done by  Mike Christopher
 
 ## Licensing
+©2016 Niantic, Inc. ©2016 Pokémon. ©1995–2016 Nintendo / Creatures Inc. / GAME FREAK inc. © 2016 Pokémon/Nintendo Pokémon and Pokémon character names are trademarks of Nintendo. The Google Maps Pin is a trademark of Google Inc. and the trade dress in the product design is a trademark of Google Inc. under license to The Pokémon Company. Other trademarks are the property of their respective owners.
+[Privacy Policy](http://www.pokemon.com/us/privacy-policy/)
 
-The source icons belong to © Nintendo/Creatures Inc./GAME FREAK Inc.
```
Comment (PokemonGoZ) [inline README.md]: Clarified in new PR.

File: README.md [hunk 2]
```diff
@@ -44,9 +44,11 @@ Outpox
 [PokemonGo-DesktopMap](https://github.com/mchristopher/PokemonGo-DesktopMap) Amazing job done by  Mike Christopher
 
 ## Licensing
+©2016 Niantic, Inc. ©2016 Pokémon. ©1995–2016 Nintendo / Creatures Inc. / GAME FREAK inc. © 2016 Pokémon/Nintendo Pokémon and Pokémon character names are trademarks of Nintendo. The Google Maps Pin is a trademark of Google Inc. and the trade dress in the product design is a trademark of Google Inc. under license to The Pokémon Company. Other trademarks are the property of their respective owners.
```
Comment (PokemonGoZ) [inline README.md]: Needed for legal reasons.

File: README.md [hunk 3]
```diff
@@ -44,9 +44,11 @@ Outpox
...
+[Privacy Policy](http://www.pokemon.com/us/privacy-policy/)
```
Comment (PokemonGoZ) [inline README.md]: Privacy Policy needed for legal reasons.

File: README.md [hunk 4]
```diff
@@ -44,9 +44,11 @@ Outpox
...
 
-The source icons belong to © Nintendo/Creatures Inc./GAME FREAK Inc.
+[PokemonGo-Bot](https://github.com/PokemonGoF/PokemonGo-Bot) is intended for academic purposes and should not be used to play the game *PokemonGo* as it violates the TOS and is unfair to the community. Use the bot **at your own risk**.
```
Comment (PokemonGoZ) [inline README.md]: Do not endorse anything for legal reasons. Use at your own risk for legal reasons. TOS statement.

File: README.md [hunk 5]
```diff
@@ -44,9 +44,11 @@ Outpox
...
 
-This project is licensed under the MIT License.
+[PokemonGoF](https://github.com/PokemonGoF) does not support the use of 3rd party apps or apps that violate the TOS.
```
Comment (PokemonGoZ) [inline README.md]: Do not endorse anything for legal reasons.

**Pull request merged**